docs/pages/deploying-vclusters/high-availability.mdx

```diff
@@ -2,13 +2,169 @@
 title: High Availability
 sidebar_label: High Availability
 ---
+By default, vCluster runs one instance of each of its components. That’s fine for many use cases, like ones that are very ephemeral (dev environments, CI/CD, etc.). But suppose your situation requires virtual clusters with more redundancy. In that case, you can use vCluster’s High Availability feature to run multiple copies of the vCluster components so that the cluster is more resistant to individual pods failing.
```

Comment:
> more resistant to individual pods failing.

more resistant to partial failures?

Part of the strategy with HA is to spread things across failure zones(if the scheduling is set correctly), so this can be used to protect against even  datacenter or region outages. Individual pod failure feels very slightly off.

Reply:
yep, will fix this